src/app/modules/item/pages/item-content/item-content.component.ts

```diff
@@ -10,4 +10,35 @@ export class ItemContentComponent {
 
   @Input() itemData?: ItemData;
 
+  values = [{
```

Comment:
I would have found better to have all this info in the template instead than in a variable (which will be very long at the end) in the ts with the code logic. It would probably have to be done with ng-template and parsing the children to extract the metadata to reuse them... probably not very clean / type-safe. 
Except if you have an idea of better solution.
(at least, these vars should be placed in a separate file (in the final perm component))

Reply:
I couldn't find a better way to pass the different values to the component than this array, but I now use templates in some parts of the component (for example the body, header or the description) so that this component can be easily repurposed.

This greatly simplified the boolean-section component, which is now part of this PR (before, it inherited the progress-section component, but now it just uses it with different templates).
They can both be tested [here](http://dev.algorea.org/branch/progress-sections/#/items/by-id/4702;parentAttempId=0;path=/details) below the item children.